ast/ddl.go

```diff
@@ -672,6 +680,22 @@ func (n *ColumnDef) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
+func (n *ColumnDef) Validate() bool {
```

Comment:
Document the `Validate()` method. Or could this be a private method (rename as `validate()`)?

Reply:
ok